tests/TimeSkippingServerFact.cs

```diff
@@ -4,16 +4,20 @@ namespace TemporalioSamples.Tests;
 using Xunit;
 
 /// <summary>
-/// The time-skipping test server can only run on x86/x64 currently.
+/// The time-skipping test server can only run on x86/x64/Arm64 currently.
 /// </remarks>
 public sealed class TimeSkippingServerFactAttribute : FactAttribute
 {
     public TimeSkippingServerFactAttribute()
     {
-        if (RuntimeInformation.ProcessArchitecture != Architecture.X86 &&
-            RuntimeInformation.ProcessArchitecture != Architecture.X64)
+        var processArchitecture = RuntimeInformation.ProcessArchitecture;
```

Comment:
I had to change this file to have some tests working on my laptop, let me know if it makes sense @cretz or you want me to revert it back